FrameworkSystem/private/standardLogging/Formatter/JsonFormatter.py

```diff
@@ -1,24 +0,0 @@
-"""
-Formatter as JSON structure
```

Comment:
Because, IIUC this class was only made to integrate the `options` parameter in the constructor:
https://github.com/DIRACGrid/DIRAC/pull/4610/files/d1a0abf8b00cbd0089346aef554787e562d413a4#diff-2aa3f51e6b7e61f78579a70a67d63f1eL13-L15

Indeed, I made this thing in `setFormat` that required any `Formatter` class to integrate the `options` parameter:
https://github.com/DIRACGrid/DIRAC/pull/4610/files/d1a0abf8b00cbd0089346aef554787e562d413a4#diff-88ba09cb1a67a7a954790596dd94e1e1L65

As the format option (`showHeaders`, `showThreadIDs`...) are now bound to `Logging` objects, I can remove the `options` parameter from the constructor of the `BaseFormatter` and, if I'm not wrong, I can also remove the `JsonFormatter` class. 
Do we need to keep it for other reasons?

Reply:
I don't think we need it. The other classes are still needed though ?

Reply:
Yes other classes are still needed.